tests/integration/jsr_tests.rs

```diff
@@ -4,87 +4,11 @@ use deno_core::serde_json::json;
 use deno_core::serde_json::Value;
 use deno_lockfile::Lockfile;
 use test_util as util;
-use test_util::itest;
 use url::Url;
 use util::assert_contains;
 use util::assert_not_contains;
-use util::env_vars_for_jsr_npm_tests;
-use util::env_vars_for_jsr_tests;
 use util::TestContextBuilder;
 
-itest!(no_module_graph_run {
-  args: "run jsr/no_module_graph/main.ts",
-  output: "jsr/no_module_graph/main.out",
-  envs: env_vars_for_jsr_tests(),
-  http_server: true,
-});
```

Comment:
@nathanwhit it might be a little challenging to move this automatically. We could maybe handle certain situations like where we see something like `jsr/node_module_graph/main.out` then seeing `jsr/node_module_graph/` being used in the args too, Then stripping that from the args and output, moving this to the json, and copying/moving the files over.